src/test/java/com/conveyal/datatools/DatatoolsTest.java

```diff
@@ -105,33 +142,13 @@ public static void setUp() throws Exception {
 
             applicationConfig.put("assets_bucket", s3Bucket);
             applicationDataConfig.put("gtfs_s3_bucket", s3Bucket);
+            applicationDataConfig.put("use_s3_storage", true);
```

Comment:
I noticed that in the server.yml file above we have this value set to `false`. There's probably a reason you're setting it to true here, but could you maybe add a comment about why?